src/StatsBase.jl

```diff
@@ -2,8 +2,8 @@ __precompile__(true)
 
 module StatsBase
     using Compat
-    import Compat.String
-    using ArrayViews
+    import Compat: String, view
```

Comment:
I think `using` is enough here, as you don't overload these functions (I know `import` was here before, and I've been guilty of this too).